.github/workflows/push-trigger.yml

```
@@ -21,26 +21,26 @@ jobs:
 
-  sonar_analysis:
-    needs: build-kotlin
-    if: "${{  github.event_name != 'pull_request' }}"
-    uses: Mahesh-Binayak/kattu/.github/workflows/gradlew-sonar-analysis.yml@sonartest
-    with:
-      SERVICE_LOCATION: kotlin/PixelPass
-    secrets:
-      SONAR_TOKEN: ${{ secrets.SONAR_TOKEN }}
-      ORG_KEY: ${{ secrets.ORG_KEY }}  # Changed from SONAR_ORGANIZATION to ORG_KEY
-      SLACK_WEBHOOK_URL: ${{ secrets.SLACK_WEBHOOK_INJI_TEAM }}
+#  sonar_analysis:
```

YAMLlint (1.37.1)

[warning] comment not indented like content

(comments-indentation)